SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -71,10 +71,14 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 
 	int GetNumRequiredClients() const { return NumRequiredClients; }
 
-	// Starts being called after PrepareTest, until it returns true.
+	// Called at the beginning of the test, use it to setup your steps. Contrary to AFunctionalTest, this will
```

Comment:
Is this ordering guaranteed? All other servers will also call `PrepareTest()` before actually starting the test?
I am worried that in some circumstances maybe the other servers can receive the PrepareTest after starting to execute steps (which would break things, right?).

Reply:
ATM this is happening even before the flow controllers register, which is why I'm not sure it's worth to change this flow, but waiting on Michael's throughts.
But yes, in theory you have problems of starting tests if it didn't run on anybody. I can still change the flow so that we know for sure that it ran, but I think it's more of a theoretical problem than something that can actually happen.